packages/amplify-cli/src/init-steps/s9-onSuccess.ts

```diff
@@ -75,6 +84,15 @@ export async function onSuccess(context: $TSContext) {
   if (!context.parameters.options.app) {
     printWelcomeMessage(context);
   }
+
+  const appId = currentAmplifyMeta?.providers?.awscloudformation?.AmplifyAppId;
+
+  if (!appId) {
+    printer.warn('You have reached your Amplify App limit:');
```

Comment:
The maximum number of apps that you can create with Amplify in this region has likely been reached:
For more information on Amplify Service Quotas, see:
https://docs.aws.amazon.com/general/latest/gr/amplify.html#service-quotas-amplify